advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>On the IAM role topic, you may also mention that any IAM role created with the managed policy <code>arn:aws:iam::aws:policy/AmazonSageMakerFullAccess</code> will  include by default the necessary read/write permissions for ECR. This policy is attached to the default "ExecutionRole" most users will setup when creating a SageMaker Noteboook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This use case specifically assumes a secure environment using a private VPC with tightly controlled access. The notebook instance that will be used to walk through these notebooks will be launched via cloud formation automation that is included with this example and explained in the README for this folder. The EFS notebook will not work at all if there is no access to a private VPC with access to EFS mount targets. All that is taken care of by the included cloud formation automation.